Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -40,6 +41,7 @@ template <typename external_spacepoint_t, typename platform_t>
 template <template <typename...> typename container_t, typename sp_range_t>
 void SeedFinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
     SeedingState& state,
+    const SeedFinderOptions&,
```

Comment:
A question here. The SeedfinderConfig does not seem to be using beam spot position. 
Is there an intention to use it in the future?

Reply:
As far as I know, there is no intention of using the beam spot position here, only in BinnedSPGroup

Reply:
ok, thanks. So this will only be used for the B field.

Reply:
If this will only be used for the B filed, do we need to pass it in `SeedFinder::createSeedsForGroup` or `SeedFinder::SeedFinder` is enough?

Reply:
I have refactored it further, to also include bField. But instead of passing it each time to these functions the options are used at construction. 
Ideally another refactor would be useful where the SeedFinder is instantiated once in clients code and then createSeedsForGroups called for every events. For another PR I guess.